src/components/FourFieldForm.js

```diff
@@ -21,5 +23,5 @@ function FourFieldForm() {
             <input type='button' value='submit' onClick={submitForm.bind(this)}></input>
           </form>
       </div>;
-}
+};
```

Comment:
This is a good setup and can even further simplified but using export default ...[your closure]... instead of declaring it and then setting it. Foe now let’s leave it. You’re doing great!